From ac37b5d8b31ffcc2afb1711e33140f05ffd1f9bb Mon Sep 17 00:00:00 2001
From: Carmen Jackson <carmenjackson@google.com>
Date: Mon, 2 Jul 2018 13:14:56 -0700
Subject: [PATCH 09/48] atrace: make 'lowmemorykiller' directory optional for
 memreclaim

The 'lowmemorykiller' directory isn't used for lmk tracing on devices that have userspace lmk.

Bug: 110999121
Test: The memreclaim option is available in Traceur on a device with
userspace lmk.

Change-Id: I5dadb5c6c55cb04b6fe079b5f10a3dc35f1ee3f1
---
 cmds/atrace/atrace.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmds/atrace/atrace.cpp b/cmds/atrace/atrace.cpp
index fe9dd568f..31e73fbec 100644
--- a/cmds/atrace/atrace.cpp
+++ b/cmds/atrace/atrace.cpp
@@ -200,7 +200,7 @@ static const TracingCategory k_categories[] = {
         { REQ,      "events/vmscan/mm_vmscan_direct_reclaim_end/enable" },
         { REQ,      "events/vmscan/mm_vmscan_kswapd_wake/enable" },
         { REQ,      "events/vmscan/mm_vmscan_kswapd_sleep/enable" },
-        { REQ,      "events/lowmemorykiller/enable" },
+        { OPT,      "events/lowmemorykiller/enable" },
     } },
     { "regulators",  "Voltage and Current Regulators", 0, {
         { REQ,      "events/regulator/enable" },
-- 
2.17.1

